vsintegration/src/FSharp.Editor/CommonHelpers.fs

```diff
@@ -0,0 +1,211 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor
+
+open System.Collections.Generic
+open System.Threading
+open System.Threading.Tasks
+open System.Runtime.CompilerServices
+
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.Classification
+open Microsoft.CodeAnalysis.Text
+
+open Microsoft.VisualStudio.FSharp.LanguageService
+open Microsoft.FSharp.Compiler.SourceCodeServices
+
+module CommonHelpers =
+    type private SourceLineData
```

Comment:
Please use single line formatting for this - multi-line is not useful - it's basically a record in any case